src/views/tutorial-view/components/next-previous/components/directional-link-box/directional-link-box.module.css

```diff
@@ -1,101 +1,41 @@
-.card {
-  border: 1px solid magenta; /** @TODO style these to spec, remove any prev imp styles necessary - https://www.figma.com/file/VD7ahvXuXWJApeGnhbW4hv/Dev-Portal?node-id=6298%3A73041 */
-  position: relative;
-  display: block;
-  background: var(--white);
-  border-radius: 5px;
-  margin: 0;
-  padding: 20px;
-}
-
-/* visually hidden */
-.hidden {
-  clip: rect(0 0 0 0);
-  clip-path: inset(50%);
-  height: 1px;
-  overflow: hidden;
-  position: absolute;
-  white-space: nowrap;
-  width: 1px;
-}
-
 .linkbox {
-  composes: card;
   display: flex;
   align-items: center;
-  width: 100%;
-  background-color: var(--white);
-  color: var(--black);
-  box-shadow: var(--token-surface-mid-box-shadow);
-  transition: box-shadow 0.3s;
+  justify-content: center;
 
-  &[data-direction='previous'] {
-    text-align: left;
-    justify-content: flex-start;
-    &:hover {
-      box-shadow: var(--token-surface-high-box-shadow);
+  @media (--medium-up) {
```

Comment:
I've refactored the media-queries here to be mobile-first. I find this much easier to reason about if it's consistent, and it also happens to DRY things up a bit.

I've also switched from the old-habit-of-mine `data-` attributes for styling to classNames. Details on the history here are in Notion, under `Code Style & Practices` > `CSS Modules`.

> Note: not to say Notion guidelines are ideal; they're just what we have. I think we're very open to changing them in any & all ways.